src/api.js

```diff
@@ -30,16 +32,32 @@ export const isSmallerRedisId = (a, b) => {
   return a1n < b1n || (a1n === b1n && number.parseInt(a2) < number.parseInt(b2))
 }
 
+const normalizeStreamMessagesReply = (/** @type {any[]} */ streamReply) => {
+  const streamReplyRes = streamReply?.map((item) => {
+    // @ts-ignore
```

Comment:
In dem File sind jetzt einige ts-ignores drin. Wollen wir die drin lassen?

Reply:
Ich hatte mit types nicht hingekriegt, wenn du ausprobieren möchtest kannst machen aber bitte mit timebox

Reply:
Das kommt, weil da jetzt die beiden Redis-Packages vermischt sind, oder? Ich schätze, um das zu wirklich zu beheben, müsste man komplett auf io-redis umstellen. Aber ich kann nicht beurteilen, ob das nötig ist. Bisher haben wir ja zumindest keine diesbezüglichen Fehler gesehen, oder?